### PR TITLE
GH-10: PHP8 compatibility.

### PR DESCRIPTION
**Pull request by nagyad**
Banched out of GH-6.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,8 @@ services:
       XDEBUG_CONFIG: "remote_enable=1 remote_host=${DOCKER_HOST_IP} remote_port=9000 idekey=PHPSTORM remote_autostart=1"
```
Comment (sergepavle) [inline docker-compose.yml]:
```suggestion
      COMPOSERVER: "--2"
      XDEBUG_CONFIG: "remote_enable=1 remote_host=${DOCKER_HOST_IP} remote_port=9000 idekey=PHPSTORM remote_autostart=1"
```

File: composer.json [hunk 2]
```diff
@@ -35,6 +34,11 @@
...
             "build/modules/contrib/{$name}": ["type:drupal-module"],
             "build/themes/contrib/{$name}": ["type:drupal-theme"]
+        },
+        "drupal-scaffold": {
+            "locations": {
+                "web-root": "./build"
+            }
         }
     },
     "autoload": {
```
Comment (sergepavle) [inline composer.json:44]: Missed `allow-plugins`

**Review by sergepavle — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (hernani) [reply docker-compose.yml]: The XDEBUG_CONFIG seems old

**Comment by nagyad** [comment]
Closing in favor of: https://github.com/drupaltest/behat-traits/pull/12

File: README.md [hunk 3]
```diff
@@ -21,8 +21,8 @@ definitions to demonstrate how to use the traits.
 
 This depends on the following software:
 
-* [PHP 7.1](http://php.net/)
-* [Drupal 8.6](https://www.drupal.org/)
+* [PHP 8.0](http://php.net/)
```
Comment (22Alexandra) [inline README.md:24]: We still support 7.4.

**Pull request closed without merging**